OpenRA.Mods.Common/Widgets/Logic/Installation/DownloadPackageLogic.cs

```diff
@@ -164,17 +164,17 @@ void ShowDownloadDialog()
 
 					Game.RunAfterTick(() => { Ui.CloseWindow(); onSuccess(); });
 				}
-				catch (Exception)
+				catch (Exception e)
 				{
-					Log.Write("install", "Extraction failed");
+					Log.Write("install", "Extraction failed: " + e.ToString());
```

Comment:
Nit: Please make this `Archive extraction failed: ...` so someone that encounters this will have something to search for in the logs if they go digging (they'll likely search for "archive extraction failed").